src/System.Memory/tests/Performance/Perf.Utf8Parser.cs

```diff
@@ -10,87 +11,513 @@ namespace System.Buffers.Text.Tests
 {
     public static partial class Utf8ParserTests
     {
-        private const int InnerCount = 100000;
+        private const int InnerCount = 100_000;
 
-        [Benchmark]
-        [InlineData("True")]
-        [InlineData("False")]
-        private static void PrimitiveParserByteSpanToBool(string text)
+        private static readonly string[] s_UInt32TextArray = new string[10]
+        {
+            "42",
+            "429496",
+            "429496729",
+            "42949",
+            "4",
+            "42949672",
+            "4294",
+            "429",
+            "4294967295",
+            "4294967"
+        };
+
+        private static readonly string[] s_UInt32TextArrayHex = new string[8]
+        {
+            "A2",
+            "A29496",
+            "A2949",
+            "A",
+            "A2949672",
+            "A294",
+            "A29",
+            "A294967"
+        };
+
+        private static readonly string[] s_Int16TextArray = new string[13]
+        {
+            "21474",
+            "2",
+            "-21474",
+            "31484",
+            "-21",
+            "-2",
+            "214",
+            "2147",
+            "-2147",
+            "-9345",
+            "9345",
+            "1000",
+            "-214"
+        };
+
+        private static readonly string[] s_Int32TextArray = new string[20]
+        {
+            "214748364",
+            "2",
+            "21474836",
+            "-21474",
+            "21474",
+            "-21",
+            "-2",
+            "214",
+            "-21474836",
+            "-214748364",
+            "2147",
+            "-2147",
+            "-214748",
+            "-2147483",
+            "214748",
+            "-2147483648",
+            "2147483647",
+            "21",
+            "2147483",
+            "-214"
+        };
+
+        private static readonly string[] s_SByteTextArray = new string[17]
+        {
+            "95",
+            "2",
+            "112",
+            "-112",
+            "-21",
+            "-2",
+            "114",
+            "-114",
+            "-124",
+            "117",
+            "-117",
+            "-14",
+            "14",
+            "74",
+            "21",
+            "83",
+            "-127"
+        };
+
+        [Benchmark(InnerIterationCount = InnerCount)]
+        [InlineData("2134567890")] // standard parse
+        [InlineData("18446744073709551615")] // max value
+        [InlineData("0")] // min value
+        [InlineData("000000000000000000001235abcdfg")]
+        [InlineData("21474836abcdefghijklmnop")]
+        private static void StringToUInt64_Baseline(string text)
```

Comment:
We have `StringToUInt64_Baseline` and `StringToUInt64`, only for UInt64, where _Baseline calls ulong.TryParse with additional arguments. We are inconsistent here. For all other numeric primitives, we just have a single test method, for instance for UInt32, we have: https://github.com/dotnet/corefx/pull/32026/files#diff-7f5a11a35cf8403d05b6acd67bc83b28R223

We should be consistent here.
Let's remove `StringToUInt64_Baseline` and rename `StringToUInt64` to match the rest.

Reply:
Shouldn't we remove StringToUInt64 and keep the baseline one, for consistency?